src/main/app/claims/ccjModelConverter.ts

```diff
@@ -0,0 +1,56 @@
+import { DraftCCJ } from 'ccj/draft/DraftCCJ'
+import { PaidAmountOption } from 'ccj/form/models/yesNoOption'
+import { RepaymentPlan as RepaymentPlanForm } from 'ccj/form/models/repaymentPlan'
+import { PaymentType } from 'ccj/form/models/ccjPaymentOption'
+import { convertDefendantDetails } from 'claims/converters/defendantDetails'
+import { RepaymentPlan } from 'claims/models/replaymentPlan'
+import { PartyDetails } from 'forms/models/partyDetails'
+import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
+import { Moment } from 'moment'
+
+function convertRepaymentPlan (repaymentPlan: RepaymentPlanForm): RepaymentPlan {
+
+  if (repaymentPlan && repaymentPlan.remainingAmount !== undefined) {
```

Comment:
Wouldn't `if (repaymentPlan && repaymentPlan.remainingAmount) {` be enough?

Reply:
done